src/mozanalysis/experiment.py

```diff
@@ -260,6 +266,14 @@ def get_time_series_data(
                 (client_id, branch), or else your results will be subtly
                 wrong.
 
+            custom_exposure_query (str): A full SQL query to be used in the main
+                query::
+
+                    WITH ...
+                    exposures AS ({custom_exposure_query})
```

Comment:
Can you document what happens if it isn't specified? How do I use the Nimbus exposure event? Ditto for exposure_metric.